examples/tutorial/tutorial_part_two.ipynb

```diff
@@ -209,6 +234,9 @@
     "\n",
     "When you register an application with multiple candidate models, the selection policy dictates which of those models to query, and if more than one model is queried, how to combine the results. Selection policies can be used to implement ensembles, to perform dynamic online model selection using multi-armed bandit algorithms (a more statistically efficient alternative to A/B testing), to estimate prediction confidence, and many other applications. Selection policies can learn from feedback and be updated in real-time directly in the serving system. For more information about selection policies in Clipper, check out the [research paper on Arxiv](https://arxiv.org/abs/1612.03079).\n",
     "\n",
+    "This diagram illustrates how the selection policy affects prediction serving with your current setup:\n",
```

Comment:
"This diagram illustrates how the EXP4 selection policy sends each prediction query to _all_ the deployed models and then combines their predictions into a single prediction that is returned to the client:"